incubator/hnc/api/v1alpha1/hnc_config_conversion.go

```diff
@@ -39,8 +41,13 @@ func (src *HNCConfiguration) ConvertTo(dstRaw conversion.Hub) error {
 	dstSpecTypes := []v1a2.TypeSynchronizationSpec{}
 	for _, st := range srcSpecTypes {
 		dt := v1a2.TypeSynchronizationSpec{}
-		dt.APIVersion = st.APIVersion
-		dt.Kind = st.Kind
+		// Hack the group from APIVersion by removing the version.
+		gv := strings.Split(st.APIVersion, "/")
+		if len(gv) == 2 {
+			dt.Group = gv[0]
+		}
```

Comment:
What if it's not?

Reply:
"v1" apiVersion will be converted to "" group (empty group).

Reply:
Maybe worth a comment